.gitignore

```diff
@@ -79,6 +79,6 @@ mould.log
 
 .env
 
-testdata
+# testdata
```

Comment:
remove this LOC

Reply:
I think we are using `testdata` folder for the files generated by test cases. We need `testdata` to  be in `.gitignore`.

Reply:
I added removeCreatedPath function in the file in which this happens.

Reply:
@saurabh-prakash removed "#testdata"